cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -72,7 +72,7 @@ spec:
     # toggle to use hostNetwork
     hostNetwork: false
   # To control where various services will be scheduled by kubernetes, use the placement configuration sections below.
-  # The example under 'all' would have all services scheduled on kubernetes nodes labeled with 'role=storage' and
+  # The example under 'all' would have all services scheduled on kubernetes nodes labeled with 'role=storage-role' and
```

Comment:
It should be `storage-node` and not `storage-role` as the `nodeAffinity` examples use.